scripts/apps/authoring/authoring/directives/AuthoringTopbarDirective.ts

```diff
@@ -45,13 +46,33 @@ export function AuthoringTopbarDirective(
             };
 
             scope.openPublishOrSendToPane = () => {
+                const activeTab = determineActiveTab(scope.item);
+
                 dispatchInternalEvent('interactiveArticleActionStart', {
                     items: [scope.item],
-                    tabs: ['send_to', 'publish'],
-                    activeTab: scope.item.flags?.marked_for_not_publication ? 'send_to' : 'publish',
+                    tabs: getTabsArray(activeTab),
+                    activeTab: activeTab,
                 });
             };
 
+            function determineActiveTab(item) {
+                if (item.flags?.marked_for_not_publication) {
+                    return 'send_to';
```

Comment:
is there a reason to set tab to `send_to` here, rather than to a first tab?

Reply:
this is something I'm not sure of.
Previously, @dzonidoo made the following changes: https://github.com/superdesk/superdesk-client-core/pull/4370, he may be able to explain this better.

Reply:
That change he made is unrelated. It was about publishing tab appearing when it wasn't supposed to appear. If you chose first available tab here all would be fine.

Reply:
got it